go/libkb/chain_link_v2.go

```diff
@@ -0,0 +1,150 @@
+package libkb
+
+import (
+	"encoding/base64"
+	"fmt"
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+)
+
+type SigchainV2Type int
+
+const (
+	SigchainV2TypeNone                        SigchainV2Type = 0
+	SigchainV2TypeEldest                      SigchainV2Type = 1
+	SigchainV2TypeWebServiceBinding           SigchainV2Type = 2
+	SigchainV2TypeTrack                       SigchainV2Type = 3
+	SigchainV2TypeUntrack                     SigchainV2Type = 4
+	SigchainV2TypeRevoke                      SigchainV2Type = 5
+	SigchainV2TypeCryptocurrency              SigchainV2Type = 6
+	SigchainV2TypeAnnouncement                SigchainV2Type = 7
+	SigchainV2TypeDevice                      SigchainV2Type = 8
+	SigchainV2TypeWebServiceBindingWithRevoke SigchainV2Type = 9
+	SigchainV2TypeCryptocurrencyWithRevoke    SigchainV2Type = 10
+	SigchainV2TypeSibkey                      SigchainV2Type = 11
+	SigchainV2TypeSubkey                      SigchainV2Type = 12
+	SigchainV2TypePGPUpdate                   SigchainV2Type = 13
+)
+
+func (t SigchainV2Type) NeedsSignature() bool {
+	switch t {
+	case SigchainV2TypeTrack, SigchainV2TypeUntrack,
+		SigchainV2TypeCryptocurrency, SigchainV2TypeAnnouncement:
```

Comment:
Actually, I stand by the code as is.  Since the cryptocurrency link doesn't prove anything about your identity, there's no reason to insist it's there.